nutrinator/generator.py

```diff
@@ -23,14 +23,15 @@ class GeneratorConfig:
 
 
 class Generator:
-    def __init__(self, config: GeneratorConfig):
+    def __init__(self, config: GeneratorConfig = GeneratorConfig()):
         self.config = config
         self.shape = (self.config.dishes,)
         # self.shape = (self.config.days, self.config.dishes)
         self.recipes = {i for i in range(config.recipes_size)} - config.taboo
         self.generator = np.random.default_rng()
 
-    def generate_days(self, size: int, previous: np.ndarray = np.array([])) -> Tuple[np.ndarray, np.ndarray]:
+    def generate_days(self, size: int = None, previous: np.ndarray = np.array([])) -> Tuple[np.ndarray, np.ndarray]:
+        size = size if size else self.config.days  # przepraszam, że to tak brzydko wygląda :c
```

Comment:
Why? Size here means number of generated solutions, not the size of single solution

Reply:
OK, I'll fix that. I wrongly assumed size was equal to days because of the shape of matrices that concidentally were equal in my case. Sory for that.